README.md

```diff
@@ -46,7 +47,34 @@ neumorphic.css is a lightweight, responsive and modern CSS framework.
 - [Copyright and License](#license)
 
 ## <a name="gettingstarted"></a>Getting started
-neumorphic.css is available via either NPM (preferred), Yarn, or a CDN.
+neumorphic.css is available via either direct installation, NPM, Yarn, or a CDN.
+
+
+### KISSest Installation
+
+1. Clone the file wherever you want via wget or curl
+
+```shell
+wget https://raw.githubusercontent.com/Webments/neumorphic.css/development/neumorphic.min.css
+```
+
+or
+
+```shell
+curl -o simplistika.css "https://raw.githubusercontent.com/Webments/neumorphic.css/development/neumorphic.min.css"
+```
+
+2. Insert the line below to your head area in your HTML file
+
+```html
+<!-- index.html (that is just an example)-->
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="neumorphic-min.css" />
+  </head>
+  <body></body>
```

Comment:
Please run a formatter before submitting any changes requested above. Automation using GitHub Actions will not be used on Webments repositories and it should be done locally before committing & pushing to remote.